tests/marqo_test.py

```diff
@@ -185,87 +196,117 @@ def warm_request(self, func, *args, **kwargs):
         for i in range(5):
             func(*args, **kwargs)
 
-    def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
-        """
-            Create a cloud index with the given name and settings.
+    def create_test_index(
+            self,
+            cloud_test_index_to_use: CloudTestIndex, open_source_test_index_name: str,
+            open_source_index_settings: dict = None, open_source_index_kwargs: dict = None,
```

Comment:
is it possible for `open_source_index_settings` to be renamed to `open_source_index_settings_dict`, just to be 100% clear how this parameter is used? 

Or at least clearly explain this parameter (and other args while you're at it) in the docstring)